README.md

```diff
@@ -1,81 +1,67 @@
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/eddiehubcommunity/support)
 
 # Community Support
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![Discord](https://img.shields.io/discord/699608417039286293?style=flat-square)](http://discord.eddiehub.org)
+
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-Raise an [issue to join the EddieHub GitHub organisation](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization).
 
-### Some vague idea of how we could turn this into a support channel.
+[Join the Discord Community](http://discord.eddiehub.org) — [Subscribe to Eddie Jaoude's YouTube](https://www.youtube.com/user/eddiejaoude)
+
```

Comment:
Possible change:
`Subscribe to Eddie Jaoude's YouTube` => `Subscribe to Eddie Jaoude's YouTube channel`